lib/rex/ui/text/shell/history_manager.rb

```diff
@@ -9,15 +9,13 @@ module Shell
 
 class HistoryManager
 
-  include Singleton
```

Comment:
Should this remain a singleton? just so we can't have more than one at a time (assuming that would be bad)

Reply:
In theory there should only be only one instance of framework running at a time, which instantiates a single history manager

Reply:
I also don't want a singleton, as there would be a single instance for the test suite. When the test suite kills all framework threads at the end of a test run, it would break the logging for the next test.

Reply:
This seems like it's resolved? 👀